e2e/node/e2e.test.ts

```diff
@@ -519,6 +524,30 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
       expect(updatedDatasetTtl).toBe(updatedDatasetAsOwnerTtl);
     });
 
+    it("can use the createContainerInContainer API to create a new container", async () => {
+      const containerNameSuggestion = "newTestContainer";
+      testContainerIriChild = `${testContainerIri}${containerNameSuggestion}/`;
+      await createContainerInContainer(testContainerIri, accessGrant, {
+        fetch: requestorSession.fetch,
+        slugSuggestion: containerNameSuggestion,
+      });
+      const parentContainer = await solidClient.getSolidDataset(
+        testContainerIri,
+        {
+          fetch: resourceOwnerSession.fetch,
+        }
+      );
+
+      const parentContainerContainsAll =
+        solidClient.getThingAll(parentContainer);
```

Comment:
```suggestion
        solidClient.getUrlAll(parentContainer, LDP.member);
```
We can be more specific in the test here. The LDP namespace may be added in the existing constant files alredy defining a bunch or IRIs to have `http://www.w3.org/ns/ldp#member`. Alternatively, `@inrupt/vocab-common-rdf` may be added as a dependency (but the former should be enough).

Reply:
This was breaking when I ran it locally. I think it's because getUrlAll is looking for a thing as the first argument and parentContainer is a Dataset.

Reply:
Oh that's right, you'd have to extract the Thing which IRI matches the container first